addons/event/tests/test_event_flow.py

```diff
@@ -1,18 +1,102 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-import datetime
+from datetime import datetime, timedelta
 
 from dateutil.relativedelta import relativedelta
 
+from odoo import Command
 from odoo.addons.event.tests.common import TestEventCommon
 from odoo.exceptions import ValidationError
+from odoo.fields import Datetime as FieldsDatetime
```

Comment:
Why not just import `fields`? That's what's normally done. Also why even alias `Datetime`?